jetty-server/src/main/config/modules/server.mod

```diff
@@ -63,7 +63,7 @@ etc/jetty.xml
 # jetty.httpConfig.compliance=RFC7230
 
 ## URI Compliance: SAFE, STRICT
-# jetty.httpConfig.uriCompliance=SAFE
+# jetty.httpConfig.uriCompliance=DEFAULT
```

Comment:
Update comment with possible values, as this may be included in documentation.

Reply:
Done.